R/morse.R

```diff
@@ -308,30 +308,52 @@ NULL
 NULL
 
 #' Survival datasets for \emph{Gammarus pulex} exposed to propiconazole
-#' during 4 days with constant exposure concentration and 10 days with time variable
+#' during 10 days with time variable
 #' exposure concentration (non-standard pulsed toxicity experiment)
 #'
 #' Survival datasets of laboratory bioassays with \emph{Gammarus pulex} 
 #' freshwater invertebrate exposed to several profiles of concentrations
-#' of one fungicide (propiconazole) (constant & pulsed). 
+#' of one fungicide (propiconazole). 
 #'
 #'
-#' @name propiconazole_varC_conc
+#' @name propiconazole_pulse_exposure_conc
```

Comment:
EDIT 2: Ok, see [commit 6c7880b
](https://github.com/pveber/morse/pull/234/commits/6c7880b8c2667669058f0b19ab4516101468f3af) if it answer your question.